bench/irmin-pack/tree.ml

```diff
@@ -724,6 +373,47 @@ module Generate_trees_from_trace (Store : Store) = struct
           aux commit_seq (i + 1)
     in
     aux commit_seq 0
+
+  let run config =
```

Comment:
I had to move this out of `Bench_suite` because of `module Stat_collector = Trace_collection.Make_stat (Store)`